bunq/sdk/context.py

```diff
@@ -57,12 +58,13 @@ class ApiContext(object):
     _PATH_API_CONTEXT_DEFAULT = 'bunq.conf'
 
     def __init__(self, environment_type, api_key, device_description,
-                 permitted_ips=None):
+                 permitted_ips=None, proxy_url=None):
```

Comment:
This function has a lot of arguments, why not reformat it to:
```python
def __init__(self, environment_type, api_key,
                   device_description,  permitted_ips=None, proxy_url=None):
```

3 args on a row or:

```python
    def __init__(
            self,
            environment_type,
            api_key,
            device_description,
            permitted_ips=None,
            proxy_url=None):
```
1 arg per line ?